tests/impl.cpp

```diff
@@ -3502,7 +3502,29 @@ result_t test_mm_srli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int64_t *_a = (const int64_t *) impl.mTestIntPointer1;
+    const int count1 = 3; /* imm8 <= 63 */
+
+    int64_t d0 = _a[0] << count1;
+    int64_t d1 = _a[1] << count1;
+
+    __m128i a = do_mm_load_ps((const int32_t *) _a);
+    __m128i c = _mm_slli_epi64(a, count1);
+#if !defined(__clang__)
```

Comment:
Why did you regard clang as a special case?

Reply:
Clang prevents `_mm_slli_epi64` from taking an imm8 which is more than 63, so we have to exclude clang from the special case